packages/devtools/cli/src/base-command.ts

```diff
@@ -206,8 +210,16 @@ export abstract class BaseCommand<T extends typeof Command = any> extends Comman
       log.config({ filter: !process.env.LOG_FILTER && !process.env.LOG_CONFIG ? LogLevel.ERROR : undefined });
     }
 
-    // Load user config file.
-    await this._loadConfig();
+    if (this.flags.target) {
+      this.log('Using remote target:', this.flags.target);
+      this._clientConfig = new Config(
```

Comment:
would you still have the file config here but just to override it with the target?

Reply:
hmm, good question. the only thing I've set in my two days of using it in this manner is the log filter since by design most of the action is happening on the remote side, but I suppose there could be things that could be applicable.

Some of this feels astonishing to me given that we use a single config file/protobuf for everything, but this isn't the only instance of that pattern and there's plenty of other things about config that I find confusing. But probably better to be consistent.